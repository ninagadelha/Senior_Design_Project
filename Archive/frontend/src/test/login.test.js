import renderer from "react-test-renderer";
import Login from "../routes/login";

jest.mock("react-router-dom");
jest.mock("axios");
jest.mock("../styles/login.css");

const renderTree = (tree) => renderer.create(tree);
describe("<Login>", () => {
  it("should render component", () => {
    expect(renderTree(<Login />).toJSON()).toMatchSnapshot();
  });
});
