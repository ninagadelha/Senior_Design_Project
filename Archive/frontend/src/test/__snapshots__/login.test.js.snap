// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login> should render component 1`] = `
<div>
  <div
    className="split-screen"
  >
    <div
      className="split-left"
    >
      <div
        className="gradient-background"
      >
        <div
          className="logo-container"
        >
          <h1
            className="title"
          >
            MySTEMGrowth Profile
          </h1>
        </div>
      </div>
    </div>
    <div
      className="split-right"
    >
      <div
        className="right-content"
      >
        <div
          className="login-card"
        >
          <h2
            className="header"
          >
            Login to your account
          </h2>
          <p>
            New here?
             
          </p>
          <div
            className="input-container"
          >
            <p
              className="input-title"
            >
              Email
            </p>
            <input
              className="text-input"
              onChange={[Function]}
              placeholder="Enter Email Address"
              type="text"
            />
          </div>
          <div
            className="input-container"
          >
            <p
              className="input-title"
            >
              Enter Password
            </p>
            <input
              className="text-input"
              onChange={[Function]}
              placeholder="Password"
              type="password"
            />
          </div>
          <div
            className="recovery-container"
          />
          <p>
            By creating account, you agree to our
             
          </p>
          <p
            className="error-message"
          />
          <button
            className="login-button"
            onClick={[Function]}
          >
            Login
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
